### PR TITLE
Adds the ability to put all small items into boots

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
It just gets rid of the list of things boots could hold and replaces it with only the weight constraint. Also, was told to do this so don't blame me for whatever reason

## Why It's Good For The Game
Boots never get used to store things since they don't store much in the way of accessible, useful items.

## Changelog
:cl:
change:made boots fit up to small items instead of a specific list
/:cl:

### CODE REVIEW

**Comment by Kerbin-Fiber** [comment]
Seems right since boots even say they can store the weight restraint

**Comment by thatguythere03** [comment]
I'm all for boots actually having widespread use. Also this isn't /tg/ so we don't have to be autistic about clothing inventory slots.

File: code/datums/components/storage/concrete/pockets.dm [hunk 1]
```diff
@@ -28,30 +28,18 @@
 	attack_hand_interact = TRUE // so the detectives would discover pockets in their hats
 
 /datum/component/storage/concrete/pockets/shoes
+	max_items = 2
 	attack_hand_interact = FALSE
```
Comment (qwertyquerty) [inline code/datums/components/storage/concrete/pockets.dm:32]: Why 2 items

Comment (MarkSuckerberg) [reply code/datums/components/storage/concrete/pockets.dm:32]: 2 shoes

Comment (thatguythere03) [reply code/datums/components/storage/concrete/pockets.dm:32]: two sides of the boot doggie
*edit
wait no fuck two boots
WHAT I MEAN IS THAT THERE IS TWO BOOTS AND EACH BOOT WOULD LOGICALLY HOLD A SINGLE ITEM

Comment (Kerbin-Fiber) [reply code/datums/components/storage/concrete/pockets.dm:32]: Two items is also the default for shoes.

Comment (Kerbin-Fiber) [reply code/datums/components/storage/concrete/pockets.dm:32]: As in, right now, we can hold two items in our shoes anyways.

**Pull request merged**